config/locales/idv/es.yml

```
@@ -292,9 +292,9 @@ es:
       technical_difficulties: Lamentablemente, debido a problemas técnicos por nuestra
         parte, tal vez no podamos verificar su identidad en estos momentos.
     warning:
-      attempts:
-        one: Por motivos de seguridad, le quedan un intento.
-        other: Por motivos de seguridad, le quedan %{count} intentos.
+      attempts_html:
+        one: Por motivos de seguridad, le quedan <strong>un intento.</strong>
```

Place period outside strong tags for consistency.
```suggestion
        one: Por motivos de seguridad, le quedan <strong>un intento</strong>.
```